.travis.yml

```diff
@@ -29,10 +29,7 @@ before_script:
   - if [ "$GRAPHQL_STYLE" = "true" ]; then (npm i -g prettier) fi
   - if [ "$GRAPHQL_STYLE" = "true" ]; then (rustup component add rustfmt clippy) fi
   - if [ "$GRAPHQL_WASM" = "true" ]; then (rustup target add wasm32-unknown-unknown) fi
-  - if [ "$GRAPHQL_WASM" = "true" ]; then (wget https://github.com/rustwasm/wasm-bindgen/releases/download/0.2.54/wasm-bindgen-0.2.54-x86_64-unknown-linux-musl.tar.gz) fi
-  - if [ "$GRAPHQL_WASM" = "true" ]; then (mkdir wasm-bindgen) fi
-  - if [ "$GRAPHQL_WASM" = "true" ]; then (tar -xzf wasm-bindgen-0.2.54-x86_64-unknown-linux-musl.tar.gz -C wasm-bindgen) fi
-  - export PATH=$PATH:$PWD/wasm-bindgen/wasm-bindgen-0.2.54-x86_64-unknown-linux-musl
+  - if [ "$GRAPHQL_WASM" = "true" ]; then (cargo install wasm-bindgen-cli --force) fi
```

Comment:
The nice bit about the former was that it kept times down. Now it's taking 6 minutes to do the build. It seems to fit in the times other builds are doing though. It'd be nice if there were a download link for the "latest" release. Not something we can fix I suppose.